configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -107,27 +118,42 @@ dma pin A03 0
 
 # feature
 feature -RX_PARALLEL_PWM
+feature -AIRMODE
 feature RX_SERIAL
+feature TELEMETRY
 feature LED_STRIP
 feature OSD
+feature ESC_SENSOR
 
 # serial
-serial 2 64 115200 57600 0 115200
+serial 0 8192 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 5 1024 115200 57600 0 115200
+serial 6 2 115200 57600 0 115200
 
 # led
 led 0 6,11::CTOBVIW:0
 led 1 7,11::CTOBVIW:0
 led 2 8,11::CTOBVIW:0
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
+
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = DPS310
```

Comment:
Not sure why you are setting this here - leaving this on the default (`AUTO`) will get the device properly.

Reply:
Fixed it. Thanks.